ansible/group_vars/bounce.yml

```diff
@@ -29,7 +29,7 @@ nginx_vhosts:
     state: "present"
     extra_parameters: |
       ignore_invalid_headers off;
-      location / {
+      location /health {
         proxy_pass http://wayland_hub;
```

Comment:
This should be `http://wayland_hub/health` I think?

Reply:
Hrm, possibly - but it seems to work. I suspect the full incoming request URI gets passed through and it works ok.